apps/merkle_patricia_tree/lib/trie.ex

```diff
@@ -20,7 +17,7 @@ defmodule MerklePatriciaTree.Trie do
   @type key :: binary() | [integer()]
 
   @empty_trie <<>>
-  @empty_trie_root_hash @empty_trie |> ExRLP.encode() |> :keccakf1600.sha3_256()
```

Comment:
Let's use method from exth_crypto

Reply:
I've done it for all apps in another branch, will send PR soon